packages/core/src/lighting/enums/DiffuseMode.ts

```diff
@@ -4,7 +4,6 @@
 export enum DiffuseMode {
   /** Solid color mode. */
   SolidColor,
-  /** Texture cube mode. */
-  Texture
-  // SphericalHarmonics
+  /** SH mode */
```

Comment:
`SH` is what? Abbreviation is not good for commets.

Reply:
+1